### PR TITLE
HW3 Commit

### PR DESCRIPTION
**Pull request by mcmcgill**
Still no luck.

### CODE REVIEW

File: HW3_McGill_Max01.Rmd [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+        obs13<-c(13,apply(data13[2:3],2,mean),apply(data13[2:3],2,sd),cor(data13[2],
+                                                                       data13[3]))
+#have the function compile a frame of the data
+  obsdf<-rbind.data.frame(obs1,obs2,obs3,obs4,obs5,obs6,obs7,obs8,obs9,obs10,obs11,obs12,
+                          obs13)
+#rename the columns of the data frame
+  colnames(obsdf)=c("Observer","dev1 mean","dev2 mean","dev1 sd","dev2 sd","correlation")
+  obsdf
+}
+
```
Comment (rsettlage) [inline HW3_McGill_Max01.Rmd]: hmm, ok, yes that is a function.  What if you were to pass in a variable for Observer and then return the stats for that observer??  You may find you write less code...

**Comment by mcmcgill** [comment]
I was having difficulty getting the loop to work over a more concise
function to create a cohesive table, and I was more focused on trying to
get the knit to work, so the longer function was my best option for the
time being, as I saw it.

I still have no luck on the pdf. Just to be sure it wasn't a problem with
my RStudio, I tried knitting other markdowns and had no problems.
Something's still not letting me get a pdf out of it, so that's all I can
turn in.

File: HW3_McGill_Max01.Rmd (hunk 1 above)
Comment (rsettlage) [reply HW3_McGill_Max01.Rmd]: Yah, I get it.  I am still hoping to help out in getting you to write less.

```
computeMeanSD<-function(x){
         temp<-c(mean(x$observer[x$dev==1]),mean(x$observer[x$dev==1]),cor(x[x$dev==1,3],x[x$dev==2,3]))
         return(temp)
 }
     summary_df<-data.frame(NA,nrow=13,ncol=3)
     for(i in 1:13){
         summary_df[i,]<-computeMeanSD(x=example_data_tidy[example_data_tidy$Observer==i,])
     }
```

File: HW3_McGill_Max01.Rmd [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+```
+This workflow begins with loading in the three datasets, specified to include only the necessary columns of data, then merging them into a single data table, first by license plate number and then by defect identification code.
+The data is then cleaned by removing NA values and columns unrelated to observations from 2017, as well as by renaming the columns to singular letter sequences that more easily facilitate manipulation.
+Next, the numbers of unique makes and models are found through the uniqueN function.
+A function is then created to deduce the most frequent entries in a column, and it is applied to both defects and the make and model columns of filtered subsets corresponding to the top five defects.
+The data, as well as translations of the specific defects provided by Google, are bound into a data frame.
+Finally, its columns are renamed and a table is produced tio represent the five most common defects, their frequencies, the most common make and model of car to carry them, and their meaning in English.
+
+\section{Appendix 1: R Code}
+\subsection{Problem 5}
```
Comment (rsettlage) [inline HW3_McGill_Max01.Rmd]: Ok, another time saver here.  IF you name your code chunks, you can then pull them back by name.  You can even reuse them.  BUT, the important point here is that you can make your Appendix super easy:

For instance, here is a code chunk
```{r Problem5_CMM_analysis, eval=T, echo=F}
   some cool code
```
```{r problem5_CMM_fig1, eval=T, echo=F, include=T, results='asis'}
   some cool figure code
```
#Appendix 1: R code  
```{r Appendix, ref.label=c("Problem5_CMM_analysis","problem5_CMM_fig1"), echo=TRUE, eval=F, tidy=TRUE, include=T}
  
```

Comment (rsettlage) [reply HW3_McGill_Max01.Rmd]: Note, I am not too concerned that you do or don't get a particular problem.  I am more focused on your being more efficient and we will keep at the actual coding throughout the semester.

**Comment by mcmcgill** [comment]
Thanks, I'll work on condensing my code for next time. How exactly does
that code chunk naming/calling work?
I'd rather leave this be and move on.

**Pull request closed without merging**